### PR TITLE
@mzikherman - Adds followArtist mutation

### PR DESCRIPTION
**Pull request by broskoski**
screenshot 2017-05-23 12 50 18

### CODE REVIEW

File: test/schema/me/follow_artist.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+          artist {
+            name
+          }
+        }
+      }
+    `
+
+    const artist = {
+      name: "Damon Zucconi",
+      birthday: "1/1/1979",
```
Comment (mzikherman) [inline test/schema/me/follow_artist.js]: really?

Comment (mzikherman) [inline test/schema/me/follow_artist.js]: cc @dzucconi is this your birthday?

**Comment by mzikherman** [comment]
LGTM! Textbook.

File: test/schema/me/follow_artist.js (hunk 1 above)
Comment (dzucconi) [inline test/schema/me/follow_artist.js]: https://www.youtube.com/watch?v=4aeETEoNfOg

Comment (broskoski) [inline test/schema/me/follow_artist.js]: Exactly

Comment (mzikherman) [inline test/schema/me/follow_artist.js]: 58cf8a1b93d8005354b1eddd-contest

**Pull request merged**